Framework/Algorithms/test/CalMuonDetectorPhasesTest.h

```diff
@@ -124,6 +124,22 @@ class CalMuonDetectorPhasesTest : public CxxTest::TestSuite {
     runExecutionTest(ws);
   }
 
+  /// Test what happens when you supply frequency in Mrad/s rather than MHz
+  void testFrequencyUnits() {
+    auto ws = createWorkspace(4, 100, "Microseconds");
+    auto calc = AlgorithmManager::Instance().create("CalMuonDetectorPhases");
+    calc->initialize();
+    calc->setChild(true);
+    calc->setProperty("InputWorkspace", ws);
+    calc->setPropertyValue("Frequency", "25"); // Mrad/s, not MHz
+    calc->setPropertyValue("DataFitted", "fit");
+    calc->setPropertyValue("DetectorTable", "tab");
+    calc->setProperty("ForwardSpectra", std::vector<int>{1, 2});
+    calc->setProperty("BackwardSpectra", std::vector<int>{3, 4});
+
+    TS_ASSERT_THROWS(calc->execute(), std::runtime_error);
```

Comment:
I think that we have the same GSL version on all the windows machines. Sometimes I've seen fitting results vary slightly between diferent boxes, with apparently the same versions of all the seemingly relevant libraries. So this issue may not be reproducible on some machines.